osu.Game.Rulesets.Mania/Mods/ManiaModCover.cs

```diff
@@ -29,6 +29,9 @@ public class ManiaModCover : ManiaModWithPlayfieldCover
 
         public override bool Ranked => false;
 
+        // Ideally we'd allow this, but it's not easy to handle due to the change in acronym from the base class.
+        public override bool ValidForFreestyleAsRequiredMod => false;
```

Comment:
Just passing commentary (not immediately actionable), but if we don't find a way forward with this, I'm fine with prefixing these clashes with a third letter for ruleset (`MMC`) or just changing the acronym to not conflict. Updating server database level should be easy, not sure about replay files and client support though.

Reply:
I'm not sure changing ruleset to "MMC" does anything. I'm not even sure I wholly understand the original comment either.

mania has three "hidden-like" mods that could potentially be required. I'm not sure how you even make allowing Fade In / Cover *work*. If someone forces HD on in the room, should mania players instead have to choose one of the three? Like I don't even know how to make sense of this.

Reply:
This was only relevant when the flag affected freemods. This mod can never be set globally with FS. Will remove.